CHANGELOG.rst

```diff
@@ -1,14 +1,18 @@
 Changes by Version
 ==================
 
-0.27.0
+0.27.0 (2018-04-18)
 -------------------
+
+- Enable development and dependency versions to be overridden from the commandline (#397) <Gary Brown>
+- Add tests for metrics exposed via prometheus (#386)  <Kevin Earls>
 - Change groupid to io.jaegertracing and remove instrumentations (#375). Sources with com.uber.jaeger have been moved to https://github.com/jaegertracing/legacy-client-java <Pavol Loffay>
-- Make some methods in HttpSender.Builder public (#376) <Pavol Loffay>
 - Avoid direct access to apache thrift from jaeger-core via transitive dependency through jaeger-thrift (#374) <Gary Brown>
+- Make some methods in HttpSender.Builder public (#376) <Pavol Loffay>
```

Comment:
Should also add "Remove reference to JAEGER_DISABLE_GLOBAL_TRACER (#379) <Charlie Sharpsteen>" - as it has been added to release note for legacy-client-java's 0.27.0 release.

Reply:
Updated.